pkg/apis/kai/v1/prometheus/prometheus.go

```diff
@@ -43,6 +43,30 @@ type Prometheus struct {
 	// ServiceMonitor defines ServiceMonitor configuration for KAI services
 	// +kubebuilder:validation:Optional
 	ServiceMonitor *ServiceMonitor `json:"serviceMonitor,omitempty"`
+
+	// ExternalPrometheusUrl defines the URL of an external Prometheus instance to use
+	// When set, KAI will not deploy its own Prometheus but will configure ServiceMonitors
+	// for the external instance and validate connectivity
+	// +kubebuilder:validation:Optional
+	ExternalPrometheusUrl *string `json:"externalPrometheusUrl,omitempty"`
+
+	// ExternalPrometheusPingConfig defines the configuration for external Prometheus connectivity validation, with defaults.
+	// +kubebuilder:validation:Optional
+	ExternalPrometheusPingConfig *ExternalPrometheusPingConfig `json:"externalPrometheusPingConfig,omitempty"`
+}
+
+type ExternalPrometheusPingConfig struct {
+	// PingsInterval defines the interval for external Prometheus connectivity validation (in seconds)
+	// +kubebuilder:validation:Optional
+	PingsInterval *int `json:"pingsInterval,omitempty"`
```

Comment:
Just "Interval", it's cleaner. You're already in the context of "ping" (or "probe"). In general, try to avoid repeating the context if you can.

A good tip is to look at the usage:
`ExternalPrometheusPingConfig.PingsInterval` vs `ExternalPrometheusPingConfig.Interval`